docroot/themes/humsci/humsci_basic/src/js/shared/linked-cards/linked-cards.js

```diff
@@ -22,10 +22,26 @@
     }
 
     // Add a click event listener to each card
-    function handleClick() {
+    function handleClick(event) {
+      // Ensure clicks on "Add to Calendar" button or title do not trigger the card click
+      if (event.target.classList.contains('addtocal') || event.target.classList.contains('addtocal-title')) {
```

Comment:
I think that any button in the card should be clickable, lets extend this to target any button element, regardless of it's class.